tests/packagedcode/data/debian/copyright/debian-slim-2021-04-07/usr/share/doc/gcc-9-base/copyright-detailed.expected.yml

```diff
@@ -1650,10 +1649,10 @@ license_detections:
           OF THIS SOFTWARE OR ITS FITNESS FOR ANY PARTICULAR PURPOSE.
     identifier: x11_lucent-0e33f797-24ab-9665-2d4b-10a0ff4d546d
   - license_expression: amd-historical
-    license_expression_spdx: LicenseRef-scancode-amd-historical
```

Comment:
These test regenerations are not a part of your PR (you've regenerated in a virtualenv which was created out of the main branch, so you've pulled in changes from there without merging it). Please do a `./configure --clean && ./configure --dev && source venv/bin/activate && scancode-reindex-licenses` whenever you change branches, as you might changes in code and licenses from another branch still in your virtualenv. Always check the diff after you do test file regeneration to check that the diff is intended and a direct consequense of the added changes. Please revert these changes.

Reply:
Resolved!